### PR TITLE
Ai kit build

### PR DESCRIPTION
**Pull request by jaenrig-ifx**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---

CONTRIBUTING.md also tells you what to expect in the PR process.

Description
* Added CY8KIT-062S2-AI board
* New board building and added to GA workflow builds

### CODE REVIEW

**Comment by ramya-subramanyam** [comment]
👍

**Review by ramya-subramanyam — APPROVED** [review]
👍

**Review by IFX-Anusha — APPROVED** [review]
👏

File: ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+freeze("$(PORT_DIR)/freeze")
+include("$(MPY_DIR)/extmod/asyncio")
+require("mip")
+require("neopixel")
```
Comment (NikhitaR-IFX) [inline ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py:4]: Do we?

**Review by NikhitaR-IFX — APPROVED** [review]

File: ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py (hunk 1 above)
Comment (jaenrig-ifx) [reply ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py:4]: The freeze we still need for the lfs files or?
The neopixel could be removed it for all boards.
Also my understanding is that mip is by default included. But we have to check that, and specially make mip work again 😄 
Let´s handle it in a separate PR.

Comment (NikhitaR-IFX) [reply ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py:4]: ok..so far yes we still need it for fs.
We can take it in another PR. All good from my side

**Pull request merged**